Improve HTTP draft wording: frames carry identifier values (#3689)
quicwg/base-drafts: draft-ietf-quic-http.md

A client that is unable to retry requests loses all requests that are in flight
when the server closes the connection.  An endpoint MAY send multiple GOAWAY
frames indicating different identifiers, but the identifier in each frame
MUST NOT be greater than the identifier in any previous frame, since clients

> might already have retried unprocessed requests on another connection.

Reviewer: I like the improvement but will highlight that this is an alternative way to present the described behaviour later on 

```
Like the server, the client MAY send subsequent GOAWAY
frames so long as the specified Push ID is strictly smaller than all previously
sent values.
```

A client that is unable to retry requests loses all requests that are in flight
when the server closes the connection.  An endpoint MAY send multiple GOAWAY

> frames indicating different identifiers, but MUST NOT increase the identifier

Reviewer 2: ```suggestion
frames indicating different identifiers, but the identifier in each frame
```

> value they carry, since clients might already have retried unprocessed requests
> on another connection.

Reviewer 2: ```suggestion
MUST NOT be greater than the identifier in any previous frame, since clients
might already have retried unprocessed requests on another connection.
```
